### ISSUE TITLE
The Whisper Hybrid encoder model with dynamic quantization functions properly, but it fails when using a full int8 model with post-training quantization.

### ISSUE THREAD

**Issue by nyadla-sys** [tensorflow/tensorflow#59716]
### 1. System information

- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
- TensorFlow installation (pip package or built from source):
- TensorFlow library (version, if pip package or github SHA, if built from source):

### 2. Code

Refer the below colab to reproduce the issue ..
https://colab.research.google.com/drive/1S_3bVlwRZkMaYvvKwtPfWlyXQLS0Bvxa?usp=sharing

```
(You can paste links or attach files by dragging & dropping them below)
- Provide links to your updated versions of the above two colab notebooks.
- Provide links to your TensorFlow model and (optionally) TensorFlow Lite Model.
```

#### Option B: Paste your code here or provide a link to a custom end-to-end colab

```
(You can paste links or attach files by dragging & dropping them below)
- Include code to invoke the TFLite Converter Python API and the errors.
- Provide links to your TensorFlow model and (optionally) TensorFlow Lite Model.
```

### 3. Failure after conversion
If the conversion is successful, but the generated model is wrong, then state what is wrong:

- Model produces wrong results and/or has lesser accuracy.
- Model produces correct results, but it is slower than expected.

### 4. (optional) RNN conversion support
If converting TF RNN to TFLite fused RNN ops, please prefix [RNN] in the title.

### 5. (optional) Any other info / logs
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by nyadla-sys** [comment]
@sachinprasadhs @mohantym @arfaian Please have a look into this issue.
Thanks in advance

**Comment by Karmadark2light** [comment]
model = "https://tfhub.dev/google/nnlm-en-dim50/2"
hub_layer = hub.KerasLayer(model, input_shape=[], dtype=tf.string, trainable=True)
hub_layer(train_examples[:3])

**Comment by pjpratik** [comment]
@nyadla-sys Thanks for reporting this issue.

Sorry for the delayed response.

I was able reproduce this issue on TF 2.11. Please find the gits [here](https://colab.research.google.com/gist/pjpratik/461189d8e5729c63a180c46278249798/59716.ipynb).

@sachinprasadhs Could you please look into this issue? Thanks.

**Comment by pkgoogle** [comment]
Hi @nyadla-sys,

Thanks for raising this issue. Are you aware of the migration to [LiteRT](https://developers.googleblog.com/en/tensorflow-lite-is-now-litert/)? This transition is aimed at enhancing our project's capabilities and providing improved support and focus for our users. As we believe this issue is still relevant to LiteRT we are moving your issue there. Please follow progress [here](https://github.com/google-ai-edge/LiteRT/issues/136).

Let us know if you have any questions. Thanks.
